src/d3d9/d3d9_device.cpp

```diff
@@ -4233,7 +4233,7 @@ namespace dxvk {
 
 
   bool D3D9DeviceEx::SupportsSWVP() {
-    return m_dxvkDevice->features().core.features.vertexPipelineStoresAndAtomics;
+    return m_dxvkDevice->features().core.features.vertexPipelineStoresAndAtomics && m_dxvkDevice->features().vk12.shaderInt8;
```

Comment:
This breaks mobile hardware, and while we don't provide first-party support I don't really see a good reason for this. Where do we use int8?

Reply:
https://github.com/doitsujin/dxvk/blob/master/src/d3d9/d3d9_swvp_emu.cpp#L225

Reply:
It only impacts D3D9Device::ProcessVertices and that was already using this feature anyway. The only difference this PR makes, is properly checking whether the feature is supported before using it.

Reply:
`Logger::err("D3D9DeviceEx::ProcessVertices: SWVP emu unsupported (vertexPipelineStoresAndAtomics)");`

This error message thrown by ProcessVertices in case of no SWVP support should also be adjusted, since it's no longer all about vertexPipelineStoresAndAtomics.

Reply:
Fair enough.